jd-geek-source/src/app/services/api.services.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable()
+
+export class ApiService {
+  constructor(private http: HttpClient) {}
+  contentTypeQuery = '?Content-type=%22application%2Fjson%22 HTTP/1.1';
+  headers = new HttpHeaders({'content-type': 'application/json'});
+
+  getTestData() {
+    var format = "JSON";
+    var apiKey = "ma6v7d7sxqmp6tbsssu7rcas"
+    const payload = JSON.stringify({apiKey, format});
```

Comment:
`payload` is unused. Are you using eslint? Eslint should have called this out.

Reply:
I thought I had eslint setup already.  I will eventually be using the payload most likely which is why I left it in.